Python/pywarpx/_libwarpx.py

```diff
@@ -18,13 +18,15 @@
 
 try:
     # --- If mpi4py is going to be used, this needs to be imported
-    # --- before libwarpx is loaded (though don't know why)
+    # --- before libwarpx is loaded, because mpi4py calls MPI_Init
     from mpi4py import MPI
     if MPI._sizeof(MPI.Comm) == ctypes.sizeof(ctypes.c_int):
         _MPI_Comm_type = ctypes.c_int
     else:
         _MPI_Comm_type = ctypes.c_void_p
+        MPI = None
```

Comment:
I am not sure this line is correct.
We should maybe add comments here: the first branch is MPICH-like flavors, the else is OpenMP-like flavors (or vice versa, need to check :) ).